crates/ir/src/structure.rs

```diff
@@ -142,6 +143,7 @@ pub struct Port {
     pub width: ExprIdx,
     pub live: Liveness,
     pub info: InfoIdx,
+    pub attrs: PortAttrs,
```

Comment:
We should store this in a SparseIndexStore that maps PortIdx to PortAttrs

Reply:
Great, exactly what I was looking for. Will do!